pyiron/lammps/control.py

```diff
@@ -208,48 +208,44 @@ def pressure_to_lammps(self, pressure, rotation_matrix):
             rotation_matrix (numpy.ndarray): The 3x3 matrix rotating from the pyiron to Lammps coordinate frame.
 
         Returns:
-            (numpy.ndarray): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
+            (list): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
         """
 
         # Get something with six elements
         if not hasattr(pressure, "__len__"):
-            if not np.isclose(np.matrix.trace(rotation_matrix), 3) and pressure == 0.0:
-                pressure = np.append(pressure * np.ones(3), pressure * np.ones(3))
-            else:
-                pressure = np.append(pressure * np.ones(3), 3 * [None])
+            # This is a hydrostatic stress. It is invariant under rotation, so no transformation needed.
+            pressure = [float(pressure), float(pressure), float(pressure), None, None, None]
```

Comment:
Maybe it's more like a style issue, but I might rather put `3*[float(pressure)]+3*[None]`